src/views/PrivacyView.vue

```diff
@@ -30,31 +30,24 @@ const navigateToHome = () => {
           </p>
           <p>E-Mail: info@remsfal.de</p>
 
-          <p class="p-card-content-header">Unseren Datenschutzbeauftragten erreichen sie unter:</p>
+          <p class="p-card-content-header">{{ $t('privacy.representative') }}</p>
           <p>
             Prof. Dr.-Ing. Alexander Stanik<br />
```

Comment:
It’s not necessary to translate the text here, but it would be better to manage and maintain all text consistently in the JSON file for easier updates and uniformity.